.ci/magician/cmd/vcr_merge.go

```diff
@@ -0,0 +1,142 @@
+package cmd
+
+import (
+	"fmt"
+	"magician/exec"
+	"magician/github"
+	"magician/source"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var vcrMergeCmd = &cobra.Command{
+	Use:   "vcr-merge",
+	Short: "Merge VCR cassettes",
+	Long: `This command is triggered in .ci/gcb-push-downstream.yml to merge vcr cassettes.
+
+	The command expects the following as arguments:
+	1. Reference commit SHA
+
+	It then performs the following operations:
+	1. Get the latest closed PR matching the reference commit SHA.
+	2. Run gsutil to list, copy, and remove the vcr cassettes fixtures.
+	`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		reference := args[0]
+		fmt.Println("Reference commit SHA: ", reference)
+		githubToken, ok := os.LookupEnv("GITHUB_TOKEN_CLASSIC")
+		if !ok {
+			fmt.Println("Did not provide GITHUB_TOKEN_CLASSIC environment variable")
+			os.Exit(1)
+		}
+		rnr, err := exec.NewRunner()
+		if err != nil {
+			fmt.Println("Error creating Runner: ", err)
+			os.Exit(1)
+		}
+
+		baseBranch := os.Getenv("BASE_BRANCH")
+		if baseBranch == "" {
+			baseBranch = "main"
+		}
```

Comment:
```suggestion
```

This fallback was added for backwards-compatibility when feature branches were first introduced - it should be fine to consider BASE_BRANCH missing as an error now.

Reply:
Done